app/models/user.rb

```diff
@@ -243,7 +243,7 @@ def normalized_suffix
   end
 
   def person_types
-    identity_person_types.presence || mpi_person_types
```

Comment:
Could we just remove the `identity_person_types` completely? Or was that one of the issues that caused a million specs to fail?

Reply:
Sure, also returning it from `mpi_person_types` to `person_types` since it's the only source now.